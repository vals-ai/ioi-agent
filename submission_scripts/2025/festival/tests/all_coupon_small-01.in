version https://git-lfs.github.com/spec/v1
oid sha256:cba3fdb78649dbcd27650a6654a76c4d061c3c0849f799365941988fa4c0a58d
size 430
