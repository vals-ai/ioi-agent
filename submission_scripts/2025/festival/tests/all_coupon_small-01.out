version https://git-lfs.github.com/spec/v1
oid sha256:cd78f3ac9766d81041604327fe482368ad4d0cb355f69b7fa8463061039c8619
size 209
